packages/moonstone/ContextualPopupDecorator/ContextualPopupDecorator.js

```diff
@@ -282,15 +282,17 @@ const Decorator = hoc(defaultConfig, (config, Wrapped) => {
 				this.positionContextualPopup();
 			}
 
-			if (this.props.open && !prevProps.open) {
-				on('keydown', this.handleKeyDown);
-				on('keyup', this.handleKeyUp);
-				this.spotPopupContent();
-			} else if (!this.props.open && prevProps.open) {
-				off('keydown', this.handleKeyDown);
-				off('keyup', this.handleKeyUp);
-				if (snapshot && snapshot.shouldSpotActivator) {
-					this.spotActivator(prevState.activator);
+			if (prevProps.open && this.state.activator) {
```

Comment:
If this component is _updated_ in any reason during a popup is open, this routine runs again.
Also, this code assumed that the second rendering right after opening a popup. Does this component is rendered twice whenever it opens a popup?